docs/website/data/integrations.yaml

```diff
@@ -1266,12 +1266,12 @@ integrations:
     software:
       - kubescape
     code:
-      - https://github.com/armosec/kubescape
-      - https://github.com/armosec/regolibrary
+      - https://github.com/kubescape/kubescape
+      - https://github.com/kubescape/regolibrary
     inventors:
-      - armo
+      - ARMO
```

Comment:
This should be lowercased, as it is a reference/key in the `organizations` object further down in the file.